services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/directory/DiagnosisKeysDateDirectory.java

```diff
@@ -63,10 +70,35 @@ public void prepare(ImmutableStack<Object> indices) {
           new DiagnosisKeysHourDirectory(diagnosisKeyBundler, cryptoProvider, distributionServiceConfig);
       return decorateHourDirectory(hourDirectory);
     });
+    this.addWritableToAll(currentIndices -> {
+      LocalDate currentDate = (LocalDate) currentIndices.peek();
+      if (currentDate.equals(diagnosisKeyBundler.getDistributionTime().toLocalDate())) {
```

Comment:
does this in effect mean _"do not add writablesToAll"_ if the associated date is the current (distributaiton date)?